deltachat-ios/Controller/ChatListController.swift

```diff
@@ -287,7 +287,12 @@ class ChatListController: UITableViewController {
         switch cellData.type {
         case .deaddrop(let deaddropData):
             safe_assert(deaddropData.chatId == DC_CHAT_ID_DEADDROP)
-            showDeaddropRequestAlert(msgId: deaddropData.msgId)
+            if dcContext.showEmails == DC_SHOW_EMAILS_ALL {
```

Comment:
this extra-handling is probably temporary. once that is settled (see recent dc and irc discussions), we may again use the same approach for all `show_emails` setting.